### PR TITLE
PlaceOfWork Model is created

### PR DESCRIPTION
**Pull request by zarina86**
Complete, check PR

### CODE REVIEW

File: db/migrate/20230926152729_create_place_of_works.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreatePlaceOfWorks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :place_of_works do |t|
+      t.datetime :Start, null: false
```
Comment (XaoGao) [inline db/migrate/20230926152729_create_place_of_works.rb:4]: @zarina86 use lower case

File: db/migrate/20230926152729_create_place_of_works.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      t.datetime :End, null: true
```
Comment (XaoGao) [inline db/migrate/20230926152729_create_place_of_works.rb:5]: @zarina86 use lower case

**Review by XaoGao — CHANGES_REQUESTED** [review]
@zarina86 fix pls

File: db/migrate/20230926152729_create_place_of_works.rb (hunk 1 above)
Comment (zarina86) [reply db/migrate/20230926152729_create_place_of_works.rb:4]: Requested changes are comleted

File: db/migrate/20230926152729_create_place_of_works.rb (hunk 2 above)
Comment (zarina86) [reply db/migrate/20230926152729_create_place_of_works.rb:5]: Done

**Pull request merged**